shop/internal/libproto/shop.proto

```diff
@@ -0,0 +1,91 @@
+syntax = "proto3";
+
+option go_package = "../libproto";
+
+package shop;
+
+service ShopService {
+    rpc CreateShop(CreateShopRequest) returns (ShopResponse);
+    rpc GetShop(GetShopRequest) returns (ShopResponse);
+    rpc EditShop(EditShopRequest) returns (ShopResponse);
+    rpc DeleteShop(DeleteShopRequest) returns (ShopResponse);
+
+    rpc CreateShopItem(CreateShopItemRequest) returns (ShopItemResponse);
+    rpc GetAllShopItems(GetAllShopItemsRequest) returns (AllShopItemsResponse);
+    rpc GetShopItem(GetShopItemRequest) returns (ShopItemResponse);
+    rpc EditShopItem(EditShopItemRequest) returns (ShopItemResponse);
+    rpc DeleteShopItem(DeleteShopItemRequest) returns (ShopItemResponse);
+
+    rpc AddBankAccount(AddBankAccountRequest) returns (AddBankAccountResponse);
+}
+
+message Shop {
+    string name = 2;
```

Comment:
why start with 2?

Reply:
forgot to change when moving `id`outside